### PR TITLE
Disable form by selection

### PR DESCRIPTION
**Pull request by tmccoy529**
Added ng-show to a few fields to hide them when a sub category is selected with attribute shouldDisableForm is added. The extra spaces were removed because I deleted a function that I had created that wasn't needed. And the one space was added because the rest of the JS functions appear to be separated by one line so for consistency.

### CODE REVIEW

File: js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js [hunk 1]
```diff
@@ -558,5 +565,9 @@
 				event.preventDefault();
 			}
 		}
+
+		function isNumeric(n) {
+			return !isNaN(parseFloat(n)) && isFinite(n);
+		  }
```
[the remark this answers is not in the document]
Comment (tmccoy529) [reply js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js]: image

Not sure whats up with the way it displays all those spaces in github but Im attaching what I see in VS Code

File: js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js [hunk 2]
```diff
@@ -11,7 +11,14 @@
 		CONSTANTS,
 		$window) {
 		var self = this;
-		var categoryId = querystringer.getAsDictionary().categoryid * 1;
+		var categoryId;
```
[the remark this answers is not in the document]
Comment (tmccoy529) [reply js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js]: I made this one line as I prefer to read inline conditionals like that but I can make it multiple lines if you'd prefer.

image

File: mockups/citysourced/citysourced-reporter-panels-ng.html [hunk 3]
```diff
@@ -60,11 +60,11 @@ <h2>Your Report Was Not Submitted</h2>
 
 		<div class="bc-citysourced-reporter" ng-show="!reporter.isDone">
 
-			<ol class="bc-citysourced-reporter-steps">
+			<ol class="bc-citysourced-reporter-steps" >
 				<li class="highlight">Choose a <br/>Report Type</li>
 				<li ng-class="{'highlight': reporter.page > 1}"
-					ng-show="reporter.shouldRequireLocation()">Enter a <br/>Location</li>
-				<li ng-class="{'highlight': reporter.page > 2}">Provide Your <br/>Contact Information</li>
+					ng-show="reporter.shouldRequireLocation() && !reporter.subCategory.shouldDisableForm">Enter a <br/>Location</li>
```
[the remark this answers is not in the document]
Comment (tmccoy529) [reply mockups/citysourced/citysourced-reporter-panels-ng.html]: Updated to OR

File: js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (tmccoy529) [reply js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js]: Okay see if thats better

File: mockups/citysourced/citysourced-reporter-panels-ng.html [hunk 4]
```diff
@@ -60,11 +60,11 @@ <h2>Your Report Was Not Submitted</h2>
 
 		<div class="bc-citysourced-reporter" ng-show="!reporter.isDone">
 
-			<ol class="bc-citysourced-reporter-steps">
+			<ol class="bc-citysourced-reporter-steps" >
 				<li class="highlight">Choose a <br/>Report Type</li>
 				<li ng-class="{'highlight': reporter.page > 1}"
-					ng-show="reporter.shouldRequireLocation()">Enter a <br/>Location</li>
-				<li ng-class="{'highlight': reporter.page > 2}">Provide Your <br/>Contact Information</li>
+					ng-show="reporter.shouldRequireLocation() || !reporter.subCategory.shouldDisableForm">Enter a <br/>Location</li>
```
Comment (martypowell) [inline mockups/citysourced/citysourced-reporter-panels-ng.html]: @tmccoy529 just waiting for you to roll this back, and then it's good to go, nice work.

**Review by martypowell — APPROVED** [review]

**Pull request merged**